apps/next/src/components/Avatar/AvatarRows.tsx

```diff
@@ -0,0 +1,63 @@
+import { Stack, styled } from '@avalabs/k2-alpine';
+import { PersonalAvatar, PersonalAvatarName } from '../PersonalAvatar';
+
+const AvatarGridRow = styled(Stack)(({ theme }) => ({
+  flexDirection: 'row',
+  justifyContent: 'center',
+  gap: theme.spacing(0.5),
+  minWidth: 'max-content',
+  width: 'max-content',
+  '&:nth-of-type(n+2)': {
+    marginTop: theme.spacing(-1.5),
+  },
+  '&:nth-of-type(even)': {
+    marginLeft: '45px',
```

Comment:
It looks like a magic number to me 😬 
Is `45px` a 50% of an avatar element maybe?
If it is then please consider to use "translate" instead
```suggestion
    translate: '50%'
```